pkg/app/ops/orphancommandcleaner/orphancommandcleaner.go

```diff
@@ -0,0 +1,85 @@
+package orphancommandcleaner
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/pipe-cd/pipe/pkg/datastore"
+	"github.com/pipe-cd/pipe/pkg/model"
+)
+
+var (
+	commandTimeOut              = 24 * time.Hour
+	interval                    = 6 * time.Hour
+	maxConsecutiveFailuresCount = 3
+)
+
+type OrphanCommandCleaner struct {
+	commandstore datastore.CommandStore
+	logger       *zap.Logger
+}
+
+func NewOrphanCommandCleaner(
+	ds datastore.DataStore,
+	logger *zap.Logger,
+) *OrphanCommandCleaner {
+	return &OrphanCommandCleaner{
+		commandstore: datastore.NewCommandStore(ds),
+		logger:       logger.Named("orphan-command-cleaner"),
+	}
+}
+
+func (c *OrphanCommandCleaner) Run(ctx context.Context) error {
+	consecutiveFailuresCount := 0
+
+	t := time.NewTicker(interval)
+	for {
+		select {
+		case <-t.C:
+			if err := c.updateOrphanCommandsStatus(ctx); err != nil {
+				c.logger.Error("failed to update orphan commands", zap.Error(err))
+				consecutiveFailuresCount++
+				if consecutiveFailuresCount == maxConsecutiveFailuresCount {
+					return err
```

Comment:
As I explained [here](https://github.com/pipe-cd/pipe/pull/1434#discussion_r558296643), we can ignore this error.
Don't have to return. And we don't need the error log too because it is already logged inside the `updateOrphanCommandsStatus` function.

Btw, It would be nice if you can add a log to show the `elapsed` time of this execution.